pane.go

```diff
@@ -88,19 +90,25 @@ func ListPanes(args []string) ([]Pane, error) {
 			return nil, errAtoi
 		}
 
-		paneIndex, errAtoi := strconv.Atoi(result[6])
+		paneId, errAtoi := strconv.Atoi(result[6])
+		if errAtoi != nil {
+			return nil, errAtoi
+		}
+
+		paneIndex, errAtoi := strconv.Atoi(result[8])
 		if errAtoi != nil {
 			return nil, errAtoi
 		}
 
 		panes = append(panes, Pane{
-			ID:          paneIndex,
+			Id:          paneId,
```

Comment:
We should not rename public entries this to don't break backward compatibility for library's users.

```suggestion
			ID:          paneId,
```